src/SFML/Audio/SoundBufferRecorder.cpp

```diff
@@ -70,9 +70,10 @@ const sfSoundBuffer* sfSoundBufferRecorder_getBuffer(const sfSoundBufferRecorder
 {
     CSFML_CHECK_RETURN(soundBufferRecorder, nullptr);
 
-    soundBufferRecorder->SoundBuffer.This = soundBufferRecorder->This.getBuffer();
+    soundBufferRecorder->SoundBuffer->This = soundBufferRecorder->This.getBuffer();
```

Comment:
You need to fill the optional at some point.
```suggestion
    soundBufferRecorder->SoundBuffer = sfSoundBuffer{soundBufferRecorder->This.getBuffer()};
```
(untested)